pkg/ruler/base/manager_metrics.go

```diff
@@ -41,67 +41,67 @@ func NewManagerMetrics(disableRuleGroupLabel bool, tf util.MetricLabelTransformF
 		metricLabelTransformer: tf,
 
 		EvalDuration: prometheus.NewDesc(
-			"cortex_prometheus_rule_evaluation_duration_seconds",
+			metricsNamespace+"_prometheus_rule_evaluation_duration_seconds",
```

Comment:
does it make sense to remove the `prometheus` prefix as well?

Reply:
The corresponding mimir metrics also contain `prometheus` so for consistency I'll leave it in.